### DISCUSSION TITLE
How good are gfn2-xtb or gfnff charges?

### DISCUSSION THREAD

**Discussion by jadlover** [grimme-lab/xtb#659] [Q&A]
In the GFN2-xtb paper it mentions that no atomic partial charges or molecular dipole moments have been used in the fit. Would it then be inappropriate to try to use the charges obtained from an xtb calculation in something like an embedding calculation where a DFT energy is calculated in the presence of the charges? How about GFN-FF charges?

**Comment by awvwgk** [comment]
We checked the quality of the GFN2-xTB dipole moments in the original publication, which give a hint on the quality of the electronic structure. While we were looking at an off-target property for GFN2-xTB the result was still reasonable.

**Reply by jadlover** [reply]
Thanks for the input!
